### PR TITLE
Fix: Improve comment/reply attachment styling 

### PR DESCRIPTION
**Pull request by Mikehrn**
There were some small UI/UX issues and differences between comments and replies on the viewer. This PR aligns them and fixes some of the UI issues (such as overlapping buttons).

Fixes https://linear.app/speckle/issue/WEB-450/improve-design-when-attaching-a-file-to-a-comment

Screenshot 2024-06-19 at 15 03 30
Screenshot 2024-06-19 at 15 03 47
Screenshot 2024-06-19 at 16 50 47
Screenshot 2024-06-19 at 16 51 00

### CODE REVIEW

File: packages/frontend-2/components/form/file-upload/ProgressRow.vue [hunk 1]
```diff
@@ -1,21 +1,18 @@
 <template>
-  <div class="bg-foundation rounded-4xl px-4 py-3 w-full max-w-full relative">
+  <div
+    class="bg-foundation rounded-lg p-2 pr-1 w-full max-w-full relative dark:bg-foundation-2 bg-neutral-100"
```
Comment (fabis94) [inline packages/frontend-2/components/form/file-upload/ProgressRow.vue:3]: 1. Use semantic colors whenever possible
2. The bg classes are clashing: there's both bg-foundation and bg-neutral-100

File: packages/frontend-2/components/viewer/comments/Editor.vue [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <!-- eslint-disable vuejs-accessibility/no-autofocus -->
 <template>
-  <div class="flex flex-col w-full max-h-28 overflow-y-auto simple-scrollbar">
+  <div
+    class="flex flex-col w-full max-h-28 overflow-y-auto imple-scrollbar p-2 sm:pb-0"
```
Comment (fabis94) [inline packages/frontend-2/components/viewer/comments/Editor.vue:4]: imple-scrollbar

File: packages/frontend-2/components/viewer/comments/Editor.vue [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <!-- eslint-disable vuejs-accessibility/no-autofocus -->
 <template>
-  <div class="flex flex-col w-full max-h-28 overflow-y-auto simple-scrollbar">
+  <div
```
Comment (fabis94) [inline packages/frontend-2/components/viewer/comments/Editor.vue:3]: double check that everything is responsive w/ working scrollbars on various screen sizes

**Review by fabis94 — CHANGES_REQUESTED** [review]

File: packages/frontend-2/components/viewer/comments/Editor.vue (hunk 2 above)
Comment (Mikehrn) [reply packages/frontend-2/components/viewer/comments/Editor.vue]: Fixed

File: packages/frontend-2/components/form/file-upload/ProgressRow.vue (hunk 1 above)
Comment (Mikehrn) [reply packages/frontend-2/components/form/file-upload/ProgressRow.vue]: Ah nice catch, that one was incidentally added, removed
 it

File: packages/frontend-2/components/viewer/comments/Editor.vue (hunk 3 above)
Comment (Mikehrn) [reply packages/frontend-2/components/viewer/comments/Editor.vue:3]: I checked on mobile/tablet and by just resizing my screen, anything else in particular you usually test?

**Review by benjaminvo — APPROVED** [review]
Sooo much better. Nice fixes!

File: packages/frontend-2/components/viewer/comments/Editor.vue (hunk 3 above)
Comment (fabis94) [reply packages/frontend-2/components/viewer/comments/Editor.vue:3]: good enough imo

**Review by fabis94 — APPROVED** [review]

**Pull request merged**